python/ml4ir/applications/ranking/tests/test_metrics.py

```diff
@@ -11,29 +11,29 @@
 GOLD_METRICS = {
     "query_count": 1408.0,
```

Comment:
@lastmansleeping So to be clear, did these and the numbers in `test_ranking_model.py` change because of the categorical embeddings addition?

Reply:
Not really. I think it changed slightly because I fixed a bug in the data loading code.